scripts/data.js

```diff
@@ -16,7 +26,17 @@ module.exports = {
     demoURL: "https://gatsbydatocmshomepage.gatsbyjs.io/",
     deploynow:
       "https://www.gatsbyjs.com/dashboard/deploynow?url=https://github.com/gatsbyjs/gatsby-starter-datocms-homepage",
+    isTypescript: false,
   },
+  // "datocms-ts": {
+  //   repo: "",
+  //   cms: "DatoCMS",
+  //   name: "datocms",
+  //   dirname: "datocms-plugin",
+  //   demoURL: "",
+  //   deploynow: "",
+  //   isTypescript: true,
```

Comment:
Actually it will still need to be there, since that's where we get the repo url from. But we won't need the `isTypescript` property anymore.

I'll patch this up here with my next commit.